### PR TITLE
674/ne definitions

### PR DESCRIPTION
**Pull request by cjhaddad**
# 💅🏼 What issue does this fix?
#674 definitions panel for NE dashboard

## 🍂 Before You Submit

* [X] Did you write tests?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Did you run `bundle exec rspec` from the root?
* [ ] Did you run `bundle exec rails rswag` from the root?
* [ ] Did you run `bundle exec rubocop` from the root?
* [X] Did you run `yarn lint` in `/client`?
* [X] Did you run `yarn test` in `/client`?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Data Migrations
* [ ] Schema Migrations
* [ ] Dependencies

## 🧳 Steps to test
the #674 has a checklist that should cover all the aspects of this ticket.

### CODE REVIEW

File: client/src/i18n/locales/en/index.json [hunk 1]
```diff
@@ -185,5 +185,17 @@
...
+  "dec": "Dec",
+
+  "definitions": "Definitions",
+  "exceededLimitDef": "The child has attended more days than they were approved for this month. You have earned the max monthly payment and will not be paid for additional days.",
+  "onTrackDef": "The child is attending most or all of their days and you will likely receive most of the subsidy payment this month.",
+  "atRiskDef": "The child has been absent frequently and you may not receive the full subsidy payment this month.",
+  "fullDaysDef": "Days with 6-10 hours attended. Approved days come from the approval letter.",
+  "hoursDef": "Days with less than 6 hours attended.",
+  "absencesDef": "You can be paid for up to five absences. Absences are included in your revenue calculations.",
+  "earnedRevenueDef2": "The revenue you have already earned this month, based on the child’s attendance. This does not include other subsidy revenue such as transportation.",
```
Comment (katelovescode) [inline client/src/i18n/locales/en/index.json:197]: Can we give this block of definitions (and probably the ones from IL) more meaningful names? `neEarnedRevenueDef` and `ilEarnedRevenueDef` or something similar to that?  Would it make sense (at some point, not necessarily in the scope of this ticket) to refactor these locale files to have a more nested structure, i.e. group universal text blocks under "universal", then nest all the nebraska definitions under a "nebraskaDefinitions" key, etc.?

**Review by katelovescode — CHANGES_REQUESTED** [review]
Function name typo, suggestion for translation key names.  Behavior-wise, do we want the clickable question mark to also sort the column at the same time? It might be hard to make it........not behave that way, but it's maybe a UX thing to consider, cc @rebeccakarasiktw would love to get your input.

**Comment by katelovescode** [comment]
@rebeccakarasiktw this is the behavior on this branch

sorting

File: client/src/i18n/locales/en/index.json (hunk 1 above)
Comment (cjhaddad) [reply client/src/i18n/locales/en/index.json:197]: yes! I like the idea of the refactor. perhaps, we can create a ticket for that. in the meantime, I will remain these to be 'ne' specific!

**Comment by cjhaddad** [comment]
> @rebeccakarasiktw this is the behavior on this branch
> 
> sorting

Yes! I know this is happening and did not want/ like this. I futzed with it for awhile and planned to bring it up at our Sunday meeting. The sorting UI is all happening via the ant-design components. The column header title we have control over, but it is wrapped in the ant-design elements for the sorting, which is ANNOYING, bc it wraps it in the a element that I can't figure out how to get access to. I don't know if this is making sense...but I will say more in our touchbase.

**Review by katelovescode — APPROVED** [review]
Amazing!  Looks great!

**Pull request merged**